### PR TITLE
superlu-dist: add v9.0.0, v9.1.0

### PR DESCRIPTION
**Pull request by balay**

### CODE REVIEW

**Review by liuyangzhuan — APPROVED** [review]

**Review by wdconinc — COMMENTED** [review]
Is it possible that the first maintainer should be [`xiaoyeli`](https://github.com/xiaoyeli) instead of [`xiaoye`](https://github.com/xiaoye)? That would seem to make more sense given the homepage and url for this package.

**Comment by balay** [comment]
> Is it possible that the first maintainer should be [`xiaoyeli`](https://github.com/xiaoyeli) instead of [`xiaoye`](https://github.com/xiaoye)? That would seem to make more sense given the homepage and url for this package.

Yeah - pushed the fix. Thanks!

Likely CI issue will be fixed with 9.0.1 update

File: var/spack/repos/builtin/packages/superlu-dist/package.py [hunk 1]
```diff
@@ -16,10 +16,12 @@ class SuperluDist(CMakePackage, CudaPackage, ROCmPackage):
 
     tags = ["e4s"]
 
-    maintainers("xiaoye", "gchavez2", "balay", "pghysels", "liuyangzhuan")
+    maintainers("xiaoyeli", "gchavez2", "balay", "pghysels", "liuyangzhuan")
 
     version("develop", branch="master")
     version("amd", branch="amd")
+    version("9.0.0", sha256="aa43d33d4b1b0f5f7b5ad7685e9a6bc25088832c6c74d2ab8f75a2c9f4e9e955")
```
Comment (wdconinc) [inline var/spack/repos/builtin/packages/superlu-dist/package.py:24]: Version typo?

Comment (balay) [reply var/spack/repos/builtin/packages/superlu-dist/package.py:24]: ops - fixed now. Thanks!

**Review by wdconinc — DISMISSED** [review]
Looks good if it passes ci builds.

File: var/spack/repos/builtin/packages/superlu-dist/package.py [hunk 2]
```diff
@@ -16,10 +16,12 @@ class SuperluDist(CMakePackage, CudaPackage, ROCmPackage):
 
     tags = ["e4s"]
 
-    maintainers("xiaoye", "gchavez2", "balay", "pghysels", "liuyangzhuan")
+    maintainers("xiaoyeli", "gchavez2", "balay", "pghysels", "liuyangzhuan")
 
     version("develop", branch="master")
     version("amd", branch="amd")
+    version("9.1.0", sha256="aa43d33d4b1b0f5f7b5ad7685e9a6bc25088832c6c74d2ab8f75a2c9f4e9e955")
+    version("9.0.0", sha256="1cb2c6dc7e8231b2ec30c1266e55e440ffca9f55527771d8df28f900dd179f9d")
```
Comment (wdconinc) [inline var/spack/repos/builtin/packages/superlu-dist/package.py]: I'm getting the reverse.
```
version("9.1.0", sha256="1cb2c6dc7e8231b2ec30c1266e55e440ffca9f55527771d8df28f900dd179f9d") # FIXME
version("9.0.0", sha256="aa43d33d4b1b0f5f7b5ad7685e9a6bc25088832c6c74d2ab8f75a2c9f4e9e955") # FIXME
```

Comment (balay) [reply var/spack/repos/builtin/packages/superlu-dist/package.py]: cascading errors :( fixed now.

**Review by wdconinc — APPROVED** [review]

**Pull request merged**